src/components/User/UserRes.vue

```diff
@@ -0,0 +1,192 @@
+<template>
+  <v-container>
+    <v-card
+      class="mb-8 mt-4 grey lighten-4"
+      style="height: 100%;"
+    >
+      <v-system-bar />
+      <v-toolbar
+        flat
+        height="65%"
+      >
+        <img
+          class="mr-3"
+          src="https://brand.uark.edu/_resources/images/shield.jpg"
+          height="55px"
+          width="55px"
+        >
+        <v-toolbar-title
+          class="text-h5"
+        >
+          My Reservations
+        </v-toolbar-title>
+        <v-btn
+          depressed
+          :right="true"
+          :absolute="true"
```

Comment:
`:right="true"` is the same thing as just writing `right`.  Change them both to `right` and `absolute` to make it more readable. 

```suggestion
          right
          absolute
```

Reply:
Done